pxr/usd/usdGeom/subset.cpp

```diff
@@ -749,7 +749,7 @@ UsdGeomSubset::ValidateSubsets(
                     valid = false;
                     if (reason) {
                         *reason += TfStringPrintf("Found overlapping index %d "
-                            "in GeomSubset at path <%s> at time %s.\n", index,
```

Comment:
I think the newline characters here were a vestige from when we had planned to use this from the Python framework. We weren't consistent about including or not including the newline though, and it somewhat complicated things when trying to write the unit tests for the `usdGeom` subset families validator. I ended up just taking them all out so that clients can decide whether or not a trailing newline makes sense.